common/data_source/azure_devops/connector.py

```
@@ -0,0 +1,469 @@
+                changed_at = committer.get("date")
+                if isinstance(changed_at, str):
+                    changed = datetime.fromisoformat(changed_at.replace("Z", "+00:00")).astimezone(timezone.utc)
+                    if not (window_start <= changed <= window_end):
+                        continue
+
+                content = fetch_file_content(client, repo_api_url, path, branch)
+                if content is None:
+                    continue
+
+                yield map_item_to_document(item, content, self.organization, self._org_url, project, repo_name, branch)
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)